src/components/simpleReview/SimpleReview.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from "react";
+import type {
+  SessionState,
+  BucketState,
+  CollectionState,
+  ValidRecord,
+} from "../../types";
+
+import * as SignoffActions from "../../plugins/signoff/actions";
+import SimpleReviewButtons from "./SimpleReviewButtons";
+import SimpleReviewHeader from "./SimpleReviewHeader";
+import PerRecordDiffView from "./PerRecordDiffView";
+
+export type StateProps = {
+  session: SessionState;
+  bucket: BucketState;
+  collection: CollectionState;
+  fetchRecords(bid: string, cid: string): Promise<Array<ValidRecord>>;
+};
+
+export type SimpleReviewProps = StateProps & {
+  approveChanges: typeof SignoffActions.approveChanges;
+  declineChanges: typeof SignoffActions.declineChanges;
+  rollbackChanges: typeof SignoffActions.rollbackChanges;
+};
+
+export default function SimpleReview({
+  bucket,
+  collection,
+  session,
+  fetchRecords,
+  approveChanges,
+  declineChanges,
+  rollbackChanges,
+}: SimpleReviewProps) {
+  const bid = bucket.data.id;
+  const cid = collection.data.id;
+  const isReviewer = session.serverInfo.user?.principals?.includes(
+    `/buckets/${bid}/groups/${cid}-reviewers`
+  );
+  const isReviewableBucket = bid?.match("-workspace");
+
+  const [records, setRecords] = useState<{
+    loading: boolean;
+    newRecords: ValidRecord[];
+    oldRecords: ValidRecord[];
+  }>({
+    loading: false,
+    newRecords: [],
+    oldRecords: [],
+  });
+
+  useEffect(() => {
+    async function getRecords() {
+      if (isReviewableBucket && isReviewer && bid && cid) {
+        setRecords({ newRecords: [], oldRecords: [], loading: true });
+        const oldRecords = await fetchRecords(
+          bid.replace("-workspace", ""),
+          cid
+        );
```

Comment:
Unfortunately this won't work everywhere.
exposing/reusing `_pickSignoffResource()` in `plugins/signoff/sagas` is the way to go.

```js
const { destination: { bucket: dbid, collection: dcid }} = _pickSignoffResource(serverInfo, bid, cid);
const oldRecords = await fetchRecords(dbid, dcid);
```